plugin/black.vim

```diff
@@ -43,11 +43,11 @@ endif
 if !exists("g:black_linelength")
   let g:black_linelength = 88
 endif
-if !exists("g:black_string_normalization")
-  if exists("g:black_skip_string_normalization")
-    let g:black_string_normalization = !g:black_skip_string_normalization
+if !exists("g:black_skip_string_normalization")
+  if exists("g:black_string_normalization")
+    let g:black_skip_string_normalization = !g:black_string_normalizationlse
```

Comment:
```suggestion
    let g:black_skip_string_normalization = !g:black_string_normalization
```

Reply:
oh that's from me messing up the selection and then paste in vim :)